examples/proc_combined_ODNP.py

```diff
@@ -0,0 +1,332 @@
+import os, time, h5py
+import pylab as plt
+from numpy import empty
+from sympy import exp as s_exp
+from sympy import symbols, Symbol
+from matplotlib.ticker import FuncFormatter
+import matplotlib.transforms as transforms
+from pyspecdata import *
+from pyspecProcScripts import *
+from pyspecProcScripts import lookup_table
+from Instruments.logobj import logobj
+logger = init_logging("info")
+Ep_signal_pathway = {'ph1':1}
+fl=fl_mod()
+filename='211213_10mM_TEMPOL_test'
+file_location = 'ODNP_NMR_comp/ODNP'
+Ep_f_slice=(0.15e3,0.4e3)
+IR_f_slice=(0.15e3,0.4e3)
+Ep_t_range = (0,0.2)
+t_range = (0,0.2)
+excluded_pathways = [(0,0)]
+R1w = 1/2.172
+nPowers = 15
+ppt = 1.5167e-3
+C=0.01
+#{{{load in log
+with h5py.File(search_filename(filename+".h5",exp_type='ODNP_NMR_comp/ODNP',unique=True),'r') as f:
+    log_grp = f['log']
+    thislog = logobj()
+    thislog.__setstate__(log_grp)
```

Comment:
a little more advanced, but I think we can write a [classmethod](https://realpython.com/instance-class-and-static-methods-demystified/) to compress these steps

Reply:
okay then for now until that classmethod is made I'll keep this here. Should that be a separate PR/new wrike task?

Reply:
no, it should be easy to implement.  I can help once everything else is resolved.